Controller/Process/Result.php

```diff
@@ -217,7 +237,7 @@ protected function _validateUpdateOrder($order, $response)
         // needed because then we need to save $order objects
         $order->setAdyenResulturlEventCode($authResult);
 
-        switch ($authResult) {
+        switch (strtoupper($authResult)) {
```

Comment:
"Received" is a code that can be expected from checkout api but not from hpp so that's why is not already here. Maybe is good idea to add it on this switch condition.

Reply:
Add the case